Resources/Maps/Ruins/corvax_kamikaze.yml

```diff
@@ -0,0 +1,10451 @@
+  - uid: 1106
+    components:
+    - type: Transform
+      pos: 12.5,6.5
+      parent: 1
+    - type: DeviceLinkSource
+      linkedPorts:
+        1107:
+        - DoorStatus: Open
+    missingComponents:
+    - Construction
+    - Destructible
+  - uid: 1107
+    components:
+    - type: Transform
+      pos: 6.5,8.5
+      parent: 1
+    - type: DeviceLinkSink
+      invokeCounter: 1
+    - type: DeviceLinkSource
+      linkedPorts:
+        1355:
+        - DoorStatus: Trigger
+    missingComponents:
+    - Construction
+    - Destructible
+- proto: Bloodpack
+  entities:
+  - uid: 1108
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.38448,-7.7195234
+      parent: 1
+  - uid: 1109
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.38448,-7.5046797
+      parent: 1
+  - uid: 1110
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.404015,-7.309367
+      parent: 1
+  - uid: 1111
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.364946,-7.0163984
+      parent: 1
+  - uid: 1112
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.364946,-6.8015547
+      parent: 1
+  - uid: 1113
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.364946,-6.664836
+      parent: 1
+  - uid: 1114
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.38448,-6.4890547
+      parent: 1
+  - uid: 1115
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 10.38448,-6.2546797
+      parent: 1
+  - uid: 1116
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.657959,-7.7195234
+      parent: 1
+  - uid: 1117
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.657959,-7.524211
+      parent: 1
+  - uid: 1118
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.657959,-7.3484297
+      parent: 1
+  - uid: 1119
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.657959,-7.074992
+      parent: 1
+  - uid: 1120
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.677494,-6.8601484
+      parent: 1
+  - uid: 1121
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.677494,-6.684367
+      parent: 1
+  - uid: 1122
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.697028,-6.508586
+      parent: 1
+  - uid: 1123
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 10.697028,-6.3328047
+      parent: 1
+- proto: Blunt
+  entities:
+  - uid: 1226
+    components:
+    - type: Transform
+      pos: 11.643209,5.029546
+      parent: 1
+- proto: BoxFolderNuclearCodes
```

Comment:
Why is this a thing?
![image](https://github.com/user-attachments/assets/fad54eb7-c927-4b61-a225-4433ed9b2387)